packages/common/src/store/pages/profile/actions.ts

```diff
@@ -88,7 +88,7 @@ export function updateCurrentUserFollows(follow = false, handle: string) {
 }
 
 export function fetchFollowUsers(
-  followerGroup: User[],
```

Comment:
Was this just very wrong before or did something else change to make the type change?

Reply:
Just very wrong! Most usages are still in Javascript, and the one Typescript one was using `any`

Reply:
gotcha, makes sense